ftw/book/profiles/default/cssregistry.xml

```diff
@@ -2,42 +2,11 @@
 <object name="portal_css">
 
   <stylesheet
-      id="++resource++ftw.book-resources/ftwbook.css"
+      id="++resource++ftw.book-resources/tinymce/tinymce_minimal.css"
```

Comment:
@busykoala is the CSS of this file also part of the SCSS file (duplication)?

I would have expected the the tinymce CSS just contains an extract of the relevant statements of the theming.scss file.
This way the tinymce CSS is only loaded within tinymce and woudl not need to be registered as CSS resource in the registry.

The other approach would be to also register the `tinymce_minimal.css` in the theming SCSS registry.

I would like to eliminate the need to register a CSS ressource completely.

Reply:
The css found in `tinymce_minimal.css` is in there only. But I indeed forgot some remains. These were removed in https://github.com/4teamwork/ftw.book/pull/112/commits/a45b574c2dbc97098ada1d1ed740be18beecf338